.gitignore

```diff
@@ -183,3 +183,5 @@ status-modules/resources
 /fiddle/node_modules/
 /fiddle/target/
 /fiddle/resources/public/images/
+
+trace.edn
```

Comment:
what is trace.edn?also we need to add config.cljs as well?

Reply:
It was added, unfortunately the file was not deleted and git kept tracking it. https://github.com/status-im/status-react/commit/591e568e373dd05125c49ca4e4d856f650c7041a

Reply:
trace.edn is also generated on some crashes with trace of error.